help_to_heat/frontdoor/schemas.py

```diff
@@ -463,7 +463,7 @@
     "E (Gas & Electricity) Ltd",
     "Ecotricity",
     "EDF",
-    "E.ON",
+    "E.ON Next",
```

Comment:
Do you also need to change the wording in `portal/models.py`?

Reply:
@JoshAdamPowell I was wondering about this, but it's listed as a Customer Portal ticket. Perhaps we should double check with client? @LadunOmideyiSoftwire

Reply:
I take it from that now being changed that that was a yes? 🙂

Reply:
@CheshireSwift we discussed at GBIS stand-up and concluded that this would be changed too since we changed it in both places when we changed Octopus to Octopus Energy :)